lib/preview/routes/index.js

```diff
@@ -30,16 +32,19 @@ const buildIndexHtml = async (sessionInfo) => {
       <body style="height:100%;display:flex;flex-direction:column;align-items:center;">
         <div>
           <h1>Domains</h1>
-          ${ domains ? getSiteList(domains, PORT) : '<p>No domains found</p>' }
+          ${ domains.length ?
+            getSiteList(domains, PORT) :
+            "<p>No domains found. You either don't have any domains set up in your portal or your personal access key is missing a scope 'cms.domains.read' required for this feature.</p>"
+          }
         </div>
         <div style="width:100%;display:flex;flex-direction:row;justify-content:space-evenly">
           <div>
             <h1>Modules</h1>
-            ${ modulesHtml ? modulesHtml : `<p>No modules found in ${dest}</p>` }
+            ${ modulesHtml ? modulesHtml : `<p>No modules found in ${fakeDest}</p>` }
```

Comment:
Can you explain to me more what we are doing with `fakeDest`? And can we change this variable name to tempDest or localDest if that is what this is referring to?

Reply:
`fakeDest` is just the destination minus the `@preview/<uuid>` to make the implementation details a little more obscured - this whole frontend needs a facelift though

Reply:
Ah, okay, gotcha - going off the "lets hide the ugly uuid from the path" comments -- understood